server/profile.py

```diff
@@ -479,4 +486,6 @@ def get_ordered_transcript(profile_uc_dict_list):
                 profile_user.has_schedule),
         course_id_to_review=course_id_to_review,
         scholarship_objs=scholarships_dict,
+        recommended_objs=recommendation_dict,
+        recommended_course_ids=recommended_course_ids,
```

Comment:
Any reason we send up the ids and the objects? I feel like we should be able to get the ids from the objects.